orchestration/dagster_orchestration/hca_orchestration/solids/stage_data.py

```diff
@@ -1,13 +1,18 @@
+import re
+
 from dagster import solid, InputDefinition, Nothing, String
 from dagster.core.execution.context.compute import AbstractComputeExecutionContext
 
 
+STAGING_BUCKET_CONFIG = {
```

Comment:
⛏️ Since this is not actually the config perhaps`STAGING_BUCKET_CONFIG` -> `STAGING_BUCKET_CONFIG_SCHEMA`